exercise_060_cluster_base/src/main/resources/application.conf

```diff
@@ -0,0 +1,87 @@
+akka {
+
+  discovery.method = config
```

Comment:
Are you using discovery anywhere? it seems you are using seed nodes below, so you're not using cluster bootstrap?

Reply:
I don't recall the exact reason for putting it here, but I know I had to do it because it broke the application when it wasn't present. I'll rerun with this line removed to check if I can reproduce the issue.